src/components/SingleArticleCard.jsx

```diff
@@ -37,8 +52,29 @@ function SingleArticleCard() {
       <p>Author: {article.author}</p>
       <p>Topic: {article.topic}</p>
       <p>Content: {article.body}</p>
+      <p>
+        Posted At: {dayjs(article.created_at).format("HH:mm:ss - DD-MM-YYYY")}
+      </p>
       <p>Votes: {article.votes}</p>
-      <p>Posted At: {article.created_at}</p>
+      <button
+        className="increase-vote-button"
+        onClick={(e) => {
+          handleChange(1);
+          e.target.disabled = true;
```

Comment:
This disabled property means that a button can only be used once - what if you have a really indecisive person that changes their mind a couple of times? 🤔 
You could get around this by saving whether or not this button is disabled in a state, then relying on that state to determine whether or not the button is disabled in the button element.